### PR TITLE
Research Component Revamp

### PR DESCRIPTION
**Pull request by krishnapandeykp**

### CODE REVIEW

File: src/app/views/research-new/research-new.component.html [hunk 1]
```diff
@@ -1,178 +1,152 @@
...
-    <div class="row">
-      <div class="col pb-3">
-        <button class="btn m-2" [class.selected]="activeTab === 'ongoing'" (click)="setActiveTab('ongoing')">Ongoing Projects</button>
-        <button class="btn m-2" [class.selected]="activeTab === 'completed'" (click)="setActiveTab('completed')">Completed Projects</button>
+  <div class="row p-5">
+    <div class="col-md-3 col-12"  >
+      <div class="p-2 bg-light" style="background-color: rgb(236, 162, 162);" >
+        <div class="shadow px-4 py-2">
+          <h1 class="my-3"><b>Research</b></h1>
+          <div class="nav-list ps-md-5 mb-3 d-flex flex-column">
```
Comment (architbubber) [inline src/app/views/research-new/research-new.component.html]: nav-list class ?? Where is this class declared?
Please remove this as I see it is not needed

File: src/app/views/research-new/research-new.component.html [hunk 2]
```diff
@@ -1,178 +1,152 @@
...
-          <tr *ngFor="let project of (activeTab === 'ongoing' ? research_projects.Ongoing : research_projects.Completed)">
-            <td>{{ project.Title }}</td>
-            <td>{{ project.Investigators }}</td>
-            <td>{{ project.Sponsor }}</td>
-          </tr>
-        </tbody>
-      </table>
+    <div class="col-md-1"></div>
+    <div class="col-md-8 col-12 align-items-center justify-content-center d-flex bg-light">
+      <video id="research-video" class="shadow m-2 border border-1 border-dark img-fluid" autoplay loop>
```
Comment (architbubber) [inline src/app/views/research-new/research-new.component.html]: id="research-video" Is this needed ?

File: src/app/views/research-new/research-new.component.html [hunk 3]
```diff
@@ -1,178 +1,152 @@
...
-</div>
-
-
-<!-- Vignettes section -->
-
-<div class="p-5 row w-100 bg-light" *ngIf="showVignettes">
-  <div class="col-12 p-4 bg-white">
-    <h1 class="text-center"><b>Research Vignettes</b></h1>
+  <!-- About Section -------->
+  <div class="col-md-10 offset-md-1 align-middle align-items-center card p-md-5 mb-5" *ngIf="showAboutSection">
```
Comment (architbubber) [inline src/app/views/research-new/research-new.component.html]: *ngIf="showAboutSection" move this to before class="

File: src/app/views/research-new/research-new.component.html [hunk 4]
```diff
@@ -1,178 +1,152 @@
...
-          <div class="vignette">
-            <img [src]="vignette.imgSrc" alt="Vignette Image" class="img-fluid">
-          </div>
-        </div>
+  <!--------- Projects section --------->
+  <div class="p-5 row w-100 bg-light" *ngIf="showProjects">
+    <div class="col-12 p-4 mb-3 bg-white">
+      <h1 class="text-center"><b>Projects</b></h1>
+    </div>
+    <div class="abbreviations col-12 shadow col-md-3 lign-middle align-items-center card p-3 pb-3" style="height:fit-content">
```
Comment (architbubber) [inline src/app/views/research-new/research-new.component.html]: lign-middle ??

File: src/app/views/research-new/research-new.component.html [hunk 5]
```diff
@@ -1,178 +1,152 @@
...
-  <div class="col-12 p-3 ">
-    <div class="row">
-      <div class="col-md-4 mb-3" *ngFor="let collaboration of collaborations">
-        <section class="card p-2 h-100">
-          <img [src]="collaboration.imgSrc" class="card-img-top img-thumbnail" style="height: 300px; max-width: 100%;" alt="Collaboration Image">
-          <div class="card-body">
-            <h5 class="card-title">{{ collaboration.title }}</h5>
-            <p class="card-text">{{ collaboration.details }}</p>
+  <!-------- Vignettes section -------->
+  <div class="p-5 row w-100 bg-light" *ngIf="showVignettes">
```
Comment (architbubber) [inline src/app/views/research-new/research-new.component.html]: Move *ngIf to first for all

File: src/app/views/research-new/research-new.component.html [hunk 6]
```diff
@@ -1,178 +1,152 @@
...
+    <div class="col-12 p-4 bg-white">
+      <h1 class="text-center"><b>Research Vignettes</b></h1>
+    </div>
+    <div class="col-12 p-md-5">
+      <div class="row" *ngFor="let vignette of research_vignettes; let i = index">
+        <div class="col-md-6 mb-3">
+          <div class="vignette-container">
+            <div class="vignette">
```
Comment (architbubber) [inline src/app/views/research-new/research-new.component.html]: class="vignette"
 class="vignette-container"
 
 where are these declared
Remove them from throughout the code

File: src/app/views/research-new/research-new.component.scss [hunk 7]
```diff
@@ -1,94 +1,138 @@
...
+  }
 
+  .col-md-3 {
+    width: 100%;
+    order: 2;
+  }
 
+  .col-md-8 {
+    width: 100%;
+    order: 1;
```
Comment (architbubber) [inline src/app/views/research-new/research-new.component.scss]: Remove this order from here and above
Instead use bootstrap classes "order-lg-1" etc wherever needed

File: docs/404.html
Comment (architbubber) [inline docs/404.html:1]: This file shouldnt be deleted

**Review by architbubber — CHANGES_REQUESTED** [review]

File: src/app/views/research-new/research-new.component.html (hunk 1 above)
Comment (krishnapandeykp) [reply src/app/views/research-new/research-new.component.html]: Done.

File: src/app/views/research-new/research-new.component.html (hunk 2 above)
Comment (krishnapandeykp) [reply src/app/views/research-new/research-new.component.html]: No, at some point it was getting used for the same ordering, but later remained unused. Deleted now.

File: src/app/views/research-new/research-new.component.html (hunk 3 above)
Comment (krishnapandeykp) [reply src/app/views/research-new/research-new.component.html]: Done for all.

File: src/app/views/research-new/research-new.component.html (hunk 4 above)
Comment (krishnapandeykp) [reply src/app/views/research-new/research-new.component.html]: done.

File: src/app/views/research-new/research-new.component.html (hunk 5 above)
Comment (krishnapandeykp) [reply src/app/views/research-new/research-new.component.html]: Updated.

File: src/app/views/research-new/research-new.component.html (hunk 6 above)
Comment (krishnapandeykp) [reply src/app/views/research-new/research-new.component.html]: removed unused classes.

File: src/app/views/research-new/research-new.component.scss (hunk 7 above)
Comment (krishnapandeykp) [reply src/app/views/research-new/research-new.component.scss]: Tried that, but it was not optimal with the requirements.

**Review by architbubber — APPROVED** [review]

**Pull request merged**